app/components/icons/assets/_Default.tsx

```diff
@@ -1,21 +1,30 @@
-import randomColor from 'randomcolor'
 import * as React from 'react'
-import Svg, { Circle, SvgProps, Text } from 'react-native-svg'
+import randomColor from 'randomcolor'
+import { View, Text } from 'react-native'
+import { tailwind } from '../../../tailwind'
+
+interface _DefaultProps extends React.SVGProps<SVGSVGElement> {
+  testID?: string
+  style?: object
+}
+
+export function _Default (symbol: string): (props: _DefaultProps) => JSX.Element {
+  return (props: _DefaultProps): JSX.Element => {
+    const { style, testID } = props
+    const height = '32px'
+    const width = '32px'
+    const bg = randomColor({ luminosity: 'bright', format: 'rgba', seed: symbol, alpha: 0.2 })
+    const text = randomColor({ luminosity: 'dark', format: 'rgba', seed: symbol, alpha: 100 })
+    const first = symbol?.substring(0, 1)
 
-export function _Default (symbol: string): (props: SvgProps) => JSX.Element {
```

Comment:
I suggest we keep the SVG element here. It's rendered differently in mobile. Also, `px` doesn't work on mobile